### ISSUE TITLE
Not able to download 'distances.npz'

### ISSUE THREAD

**Issue by NimmyShaji** [antonisa/lang2vec#14]
Not able to go to the link provided under the title, 'Retrieving pre-computed distances', so as to download distances.npz

**Comment by antonisa** [comment]
The link still seems to work on my end, were you able to resolve this?

**Comment by NimmyShaji** [comment]
Yes, I made someone else download it for me. It still doesn't work in my
system though. The file is no longer the issue now. I'm having trouble with
using the distances module in my python project. I am currently working on
figuring that out. Thanks though.
Regards,
Nimmy Susan Shaji

**Comment by busracantaoglu** [comment]
Not able to go to the link provided under the title, 'Retrieving pre-computed distances', so as to download distances.npz
